### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by o-msh**
### Popup

[Demo](https://o-msh.github.io/kottans-frontend/practice/popup/dist/) |
[Code base](https://github.com/o-msh/kottans-frontend/tree/master/practice/popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/o-msh/popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+:focus {
+  outline: 1px solid #aeb5ff;
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/popup/css/style.css]: Make it bolder/stronger.

File: submissions/o-msh/popup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+}
+
+.header {
+  font: small arial,sans-serif;
```
Comment (A-Ostrovnyy) [inline submissions/o-msh/popup/css/style.css]: `font` is old property, better use extended syntax for this.

File: submissions/o-msh/popup/css/style.css (hunk 1 above)
Comment (o-msh) [reply submissions/o-msh/popup/css/style.css]: Ok

**Comment by o-msh** [comment]
@A-Ostrovnyy Thanks for review. Can you watch again?

**Comment by A-Ostrovnyy** [comment]
Nice job!

**Pull request merged**